.husky/pre-push

```diff
@@ -4,13 +4,12 @@
 validate_branch_name() {
   local branch=$(git rev-parse --abbrev-ref HEAD)
 
-  if [[ $branch =~ ^renovate ]]
-  then
-      exit 0
+  if echo $branch | grep -Eq ^renovate; then
```

Comment:
In the template you linked, they surrounded the regex with single quotes. I don't know if that makes any difference.